modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -223,10 +239,11 @@ endif::[]
 
 Persistent volumes are created by default when deploying the container.
 
-However, it is recommended that the repositories and the database for {productname} are stored on separate storage devices.
+However, it is recommended that the volumes are stored on one or more separate storage devices.
 Such a setup helps avoid data loss in production environments.
+This can be done after container deployment.
 
-Storage devices must be set up prior to deploying the container.
+Storage devices best should be set up after first deploying the container.
 For more details, see xref:installation-and-upgrade:container-management/persistent-container-volumes.adoc[Persistent Container Volumes].
 
 {productname} requires three different volumes:
```

Comment:
Didn't I mention this before? This is not right. We have plenty (28 currently) of the volumes. Please remove this.